### ISSUE TITLE
Disable push notification

### ISSUE THREAD

**Issue by Tealk** [nextcloud/calendar#4034]
### Is your feature request related to a problem? Please describe.

If you have installed the Nextcloud app and set up the calendar on your phone, you will get calendar notifications 2 times.

### Describe the solution you'd like

Turn off push notifications for the calendar

**Comment by major-mayer** [comment]
Exactly the problem that I also have. 
It would be really convenient if this could simply be added to the activity notification settings:
![grafik](https://user-images.githubusercontent.com/17907799/182139490-72235fc3-3217-425b-9719-b649919b6989.png)
Or if this doesn't fit, to the settings of the calendar app itself.

**Comment by tcitworld** [comment]
Admins have a toggle in groupware settings to enable/disable push notifications for events. I renamed the title of the issue accordingly.

**Comment by major-mayer** [comment]
Ah, i haven't known this yet, thanks for the hint.
But it would be really nice, if this could be disabled per user.

**Comment by Tealk** [comment]
Yes that would be better per user, if I turn this off in general a lynch mob will soon be following me around

**Comment by floriangosse** [comment]
I think we actually need notification settings per device/notification target. This way you can turn of the notifications on your phone but keep them for your browser.

**Comment by rotanid** [comment]
would be nice.

is related to https://github.com/nextcloud/server/issues/42659
